src/translations/en.json

```diff
@@ -2120,7 +2120,7 @@
           "manage_labels": "Manage labels",
           "no_labels": "You don't have any labels",
           "introduction": "Labels can help you organize your areas, devices and entities. They can be used to filter in the UI, or use them as a target in automations.",
-          "introduction2": "Go to the area, device or entity you want to add a label to, and click on the edit button to assign labels to them.",
+          "introduction2": "Go to the area, device or entity you want to add a label to, and select the edit button to assign labels to them.",
```

Comment:
I think "press" is more appropriate here as it is a button and not an option from a list

Reply:
OK for me.